server/channels/app/post_metadata.go

```diff
@@ -800,7 +800,7 @@ func (a *App) parseLinkMetadata(requestURL string, body io.Reader, contentType s
 		image, err := parseImages(io.LimitReader(body, MaxMetadataImageSize))
 		return nil, image, err
 	} else if strings.HasPrefix(contentType, "text/html") {
-		og := a.parseOpenGraphMetadata(requestURL, body, contentType)
+		og, _, _ := a.parseOpenGraphMetadata(requestURL, body, contentType)
```

Comment:
There's a potential nil dereference and panic on the next lines since og can now be nil.

Reply:
Ah, nice catch, thank you, fixed.